tests/rptest/tests/schema_registry_test.py

```diff
@@ -928,3 +944,52 @@ def clean_nodes(selfself, nodes):
         svc = StressTest(self._ctx)
         svc.start()
         svc.wait()
+
+    @cluster(num_nodes=3)
+    def test_protobuf(self):
+        """
+        Verify basic protobuf functionality
+        """
+
+        self.logger.info("Posting failed schema should be 422")
```

Comment:
Rogue log line?

Reply:
> Rogue log line?

No, that's to describe the following test. Any suggestions on how to make it clearer?

Reply:
Ah, OK.  My brain parsed this as "Posting failed!  Schema should be 422".

Probably doesn't matter.